dev-docs/publisher-api-reference/setConfig.md

```diff
@@ -745,6 +745,7 @@ The targeting key names and the associated prefix value filtered by `allowTarget
 | CACHE_ID | `hb_cache_id` | yes | Network cache ID for AMP or Mobile |
 | CACHE_HOST | `hb_cache_host` | yes | |
 | ADOMAIN | `hb_adomain` | no | Set to bid.meta.advertiserDomains[0]. Use cases: report on VAST errors, set floors on certain buyers, monitor volume from a buyer, track down bad creatives. |
+| ACAT | `hb_acat` | no | Set to bid.meta.primaryCatId. Optional category targeting key that can be sent to ad servers that stores the value of the Primary IAB category ID if present. |
```

Comment:
Can we add some description in the section below indicating why a publisher may want to pass this category, specifically, if they are attempting category exclusion with a ad server order or programmatic direct deal on another ad slot.

Reply:
@jlquaccia ^

Reply:
@patmmccann @fowler446 no prob, just updated the doc for this with another commit.